internal/trace/metrics.go

```diff
@@ -108,14 +108,14 @@ var (
 		Name:        "cloudsqlconn/bytes_sent",
 		Measure:     mBytesSent,
 		Description: "The number of bytes sent to Cloud SQL",
-		Aggregation: view.Sum(),
+		Aggregation: view.LastValue(),
```

Comment:
This is a breaking change. Perhaps we could keep `Sum` and instead use https://pkg.go.dev/sync/atomic#Int64.Swap in the reporting goroutine to reset the byte counter to 0? What do you think?

Reply:
AFAIUI, `Sum()` and `LastValue()` decide how OpenCensus handles values provided to `stats.Record()`. They do not change the output that Prometheus receives. Prometheus still gets one number that is the total number of bytes sent or received over a connections since it was established.

Reply:
For people who export to Cloud Monitoring, this setting controls how the initial metric descriptor is created. Sum maps to a cumulative metric, and last value maps to a gauge metric. Changing the aggregation will cause any additional writes to the metric to fail.

I think we can use use atomic.SwapInt64 to capture the existing count and reset the count to zero. And then we avoid the breaking change here.

``` golang
func (i *instrumentedConn) reportCounters() {
  bytesRead := atomic.SwapInt64(&i.bytesRead, 0)
  bytesWritten := atomic.SwapInt64(&i.bytesWritten, 0)
  trace.RecordBytesReceived(context.Background(), bytesRead, i.connName, i.dialerID)
  trace.RecordBytesSent(context.Background(), bytesWritten, i.connName, i.dialerID)
}
```

Reply:
Oh wow :/

Ok, then `Sum() + SwapInt64()` it is.